dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
I would favor not having the ``s``. Grammatically speaking, the situation we have is that the subject of the sentence is implied, and most of the time when a sentence's subject is implied, the subject is "You". In this case, I interpret it as  saying "You get whether a generalized coordinate is cyclic", which is why I prefer not having the ``s``.

On the other hand, if the implied subject is "This function" then having the ``s`` would make sense, because you'd be saying "This function gets whether a generalized coordinate is cyclic".

But I prefer the former over the latter.